Functions/src/main/java/com/epam/engx/cleancode/functions/task4/Order.java

```diff
@@ -10,19 +11,16 @@
     private List<Product> products;
 
     public Double getPriceOfAvailableProducts() {
+        //We don't need here Iterator, is it?
```

Comment:
You can use comments to your PR (like one you reading right now) but not comments in code